fenix/app/src/main/res/drawable/ic_bookmarks_search.xml

```diff
@@ -7,15 +7,8 @@
     android:height="24dp"
     android:viewportWidth="24"
     android:viewportHeight="24">
-    <path
-        android:name="path"
-        android:fillColor="@color/photonWhite"
-        android:pathData="M 2 0 L 22 0 C 22.53 0 23.039 0.211 23.414 0.586 C 23.789 0.961 24 1.47 24 2 L 24 22 C 24 22.53 23.789 23.039 23.414 23.414 C 23.039 23.789 22.53 24 22 24 L 2 24 C 1.47 24 0.961 23.789 0.586 23.414 C 0.211 23.039 0 22.53 0 22 L 0 2 C 0 1.47 0.211 0.961 0.586 0.586 C 0.961 0.211 1.47 0 2 0"
-        android:strokeWidth="1" />
-    <path
-        android:pathData="m7.225,11.324 l-2.86,-2.375a1.207,1.207 0,0 1,-0.378 -1.309,1.205 1.205,0 0,1 1.074,-0.835l3.697,-0.239 0.501,-0.375 1.364,-3.427A1.209,1.209 0,0 1,11.75 2c0.499,0 0.941,0.3 1.127,0.763v0.001l1.364,3.427 0.501,0.375 3.697,0.239c0.498,0.032 0.92,0.36 1.074,0.835 0.155,0.476 0.006,0.99 -0.378,1.309L16.28,11.32l-0.18,0.592 0.91,3.575c0.123,0.485 -0.059,0.986 -0.463,1.279a1.202,1.202 0,0 1,-1.359 0.046l-3.127,-1.977h-0.621l-3.126,1.977c-0.426,0.266 -0.96,0.247 -1.361,-0.045a1.205,1.205 0,0 1,-0.463 -1.279l0.908,-3.566 -0.173,-0.598z"
-        android:fillColor="@color/photonBlack"/>
-    <path
-        android:pathData="m5.782,20.5 l-0.8,-0.8v-1.95a0.75,0.75 0,0 0,-1.5 0v1.75a2.5,2.5 0,0 0,2.5 2.5h11.535a2.5,2.5 0,0 0,2.5 -2.5v-1.75a0.75,0.75 0,0 0,-1.5 0v1.95l-0.8,0.8H5.782z"
-        android:fillColor="@color/photonBlack"/>
+  <path
+      android:pathData="M4.365,8.949L7.225,11.324L7.398,11.922L6.49,15.488C6.367,15.972 6.549,16.474 6.953,16.767C7.354,17.059 7.888,17.078 8.314,16.812L11.44,14.835H12.061L15.188,16.812C15.387,16.938 15.612,17 15.835,17C16.085,17 16.334,16.922 16.547,16.766C16.951,16.473 17.133,15.972 17.01,15.487L16.1,11.912L16.28,11.32L19.135,8.949C19.519,8.63 19.668,8.116 19.513,7.64C19.359,7.165 18.937,6.837 18.439,6.805L14.742,6.566L14.241,6.191L12.877,2.764V2.763C12.691,2.3 12.249,2 11.75,2C11.251,2 10.809,2.3 10.623,2.764L9.259,6.191L8.758,6.566L5.061,6.805C4.563,6.837 4.141,7.165 3.987,7.64C3.832,8.116 3.981,8.63 4.365,8.949ZM4.982,19.7L5.782,20.5H17.717L18.517,19.7V17.75C18.517,17.336 18.853,17 19.267,17C19.681,17 20.017,17.336 20.017,17.75V19.5C20.017,20.881 18.898,22 17.517,22H5.982C4.601,22 3.482,20.881 3.482,19.5V17.75C3.482,17.336 3.818,17 4.232,17C4.646,17 4.982,17.336 4.982,17.75V19.7Z"
+      android:fillColor="@color/mozac_ui_icons_fill"
```

Comment:
Overall in the app `android:fillColor="?attr/textPrimary" ` is used for the automatic support for dark/light colours, but it doesn't work in this case because drawables are [converted](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/components/Core.kt#217) to `Bitmap` to be stored in `SearchEngine`, and then converted back to drawables to be used in `TextMenuCandidate`.

I am not sure why this double conversion was created, maybe should remove as a follow up.